.coveragerc

```diff
@@ -1,4 +1,5 @@
 [run]
 omit =
+    **/_*.py
```

Comment:
I'm not 100% sure about this as, sometimes, code does end up in, e.g., ``__init__.py`` files and lack of testing them would be silent.